gmso/core/topology.py

```diff
@@ -989,3 +1155,123 @@ def __repr__(self):
     def __str__(self):
         """Return custom format to represent topology as a string."""
         return f"<Topology {self.name}, {self.n_sites} sites, id: {id(self)}>"
+
+    def _pandas_from_parameters(
+        self, df, parameter, labels=None, unyts_bool=True
```

Comment:
we should add some simple docstring for this method too

Reply:
Done!